desktop-widgets/filterwidget2.h

```diff
@@ -13,7 +13,8 @@ namespace Ui {
 
 class FilterWidget2 : public QWidget {
     Q_OBJECT
-    FilterWidget2(QWidget *parent);
+public:
+    explicit FilterWidget2(QWidget *parent = 0);
```

Comment:
this really should be just squashed into the previous commit. After it's switched to tab indent, that is